runner/options.go

```diff
@@ -142,7 +144,8 @@ func ParseOptions() *Options {
 		len(options.ZoomEye) == 0 &&
 		len(options.Netlas) == 0 &&
 		len(options.CriminalIP) == 0 &&
-		len(options.Publicwww) == 0 {
+		len(options.Publicwww) == 0 &&
+		len(options.HunterHow) == 0 {
```

Comment:
@xm1k3 , I think we can use newly added generic comparison helper function here https://github.com/projectdiscovery/utils/issues/90

Ex:
```go
genericutils.EqualsAll(0,len(options.ZoomEye),len(options.Netlas))
```